src/pages/debrief/__tests__/debrief.spec.ts

```diff
@@ -161,13 +161,18 @@ describe('DebriefPage', () => {
     });
 
     describe('endDebrief', () => {
-      it('should navigate to PassFinalisationPage when passed = true', () => {
-        component.passed = true;
+      it('should navigate to PassFinalisationPage when outcome = pass', () => {
+        component.outcome = 'pass';
         component.endDebrief();
         expect(navController.push).toHaveBeenCalledWith('PassFinalisationPage');
       });
-      it('should navigate to OfficePage when passed = false', () => {
-        component.passed = false;
+      it('should navigate to BackToOfficePage when outcome = fail', () => {
+        component.outcome = 'fail';
+        component.endDebrief();
+        expect(navController.push).toHaveBeenCalledWith('BackToOfficePage');
+      });
+      it('should navigate to the BackToOfficePage when outcomes = terminated', () => {
+        component.outcome = 'terminated';
```

Comment:
Could we have the values as a enum Outcome.Pass , Outcome.Fail and Outcome.Terminated, it would feel more robust then using strings

Reply:
As discussed, leaving as-is for now, as this is an interim solution until we play the result calculation story which will require the status for debrief to be in the store.